packages/spark-core/settings/_colors.scss

```diff
@@ -2,46 +2,40 @@
 // Colors
 // ==================================================================
 
-// TODO: fall
-// Greens
-$green-light: rgb(211, 239, 141);
-$green-medium: rgb(108, 159, 46);
-$green-dark: rgb(60, 120, 20);
-
-// Blues
-$blue-light: rgb(195, 211, 227);
-$blue-medium: rgb(32, 129, 191);
-$blue-dark: rgb(12, 69, 105);
-
-// Purples
-$purple-medium: rgb(85, 19, 135);
-$purple-dark: rgb(68, 35, 93);
-
-// Reds
-$red-light: rgb(250, 204, 196);
-$red-medium: rgb(181, 18, 27);
-$red-dark: rgb(140, 24, 29);
-
-// Oranges
-$orange-medium: rgb(255, 155, 0);
-$orange-dark: rgb(230, 120, 0);
-
-// Neutrals
-$gray-light: rgb(229, 229, 229);
-$gray-medium: rgb(170, 170, 170);
-$gray-dark: rgb(90, 90, 90);
-$black: rgb(34, 34, 34);
+// Primary
+$red-real: rgb(200, 16, 46);
```

Comment:
I don't understand the names in here. `red-real` stands out. All of the others are just one word colors with tint percentage numbers. That makes sense to me. The other one that doesn't fit is `gray-light`

Reply:
@yodasw16  I responded to you via Chat.